site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Kubernetes: there is an app for that"
+image: /img/posts/2020/11/19/survey-results-banner.png
+excerpt: Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems
+author_name: Milan Klanjsek
+author_avatar: /img/contributors/milan-klanjsek.jpg
+categories: ['kubernetes']
+tags: ['octant', 'release']
+---
+
+<h1>Kubernetes: there is an app for that</h1>
+
+Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems. The command-line version will still be available for the next couple of releases to allow a smooth transition for everyone,after which we will be 100% focused on the desktop version.
+
+![Octant Electron applications view](/img/posts/2021/02/19/electron-applications.png)
+
+
+To create a desktop version of Octant we used Electron. Electron is a popular cross-platform framework used by many well known desktop applications. Electron is a mature technology built on top of Chromium engine with a solid production environment and great community support. Leveling up to a desktop application opens the doors for many new and exciting things Octant can leverage in the future to improve the overall user experience.
```

Comment:
Comma: `To create a desktop version of Octant, we used Electron`

Hyphen: `well-known` Generally descriptors to nouns use the hyphen.